io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,49 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "Pdf4QtViewerProfi",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "cleanup": [
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "mkdir -p /app/lib/pdf4qt",
+                "mv -f /app/lib/*Plugin.so*  /app/lib/pdf4qt/",
+                "cp /app/share/icons/io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.svg /app/share/icons/io.github.JakubMelka.Pdf4qt.svg",
```

Comment:
svg go in `/app/share/icons/hicolor/scalable/apps`

Reply:
Fixed.